lib/index.js

```diff
@@ -1,6 +1,7 @@
 var Assert = require('assert');
 var Hoek = require('hoek');
 var Joi = require('joi');
+var Schema = require('mongoose').Schema
```

Comment:
Good stuff. Not sure which version of Joigoose you're using, but it might be worth making sure you're using the latest version if you use nested schemas. I've done a lot of work around it recently.